src/modules/gimbal/input_rc.cpp

```diff
@@ -112,7 +112,7 @@ InputRC::UpdateResult InputRC::_read_control_data_from_subscription(ControlData
 	// Detect a big stick movement
 	const bool major_movement = [&]() {
 		for (int i = 0; i < 3; ++i) {
-			if (fabsf(_last_set_aux_values[i] - new_aux_values[i]) > 0.25f) {
+			if (!PX4_ISFINITE(_last_set_aux_values[i]) || (fabsf(_last_set_aux_values[i] - new_aux_values[i]) > 0.25f)) {
```

Comment:
If it's NAN then that's a major movement?

Shouldn't it be:
```suggestion
			if (PX4_ISFINITE(_last_set_aux_values[i]) && (fabsf(_last_set_aux_values[i] - new_aux_values[i]) > 0.25f)) {
```

Reply:
This was added to force an initial update if your AUX channel happens to start close to zero.

https://github.com/PX4/PX4-Autopilot/pull/20013/files#diff-36d4995a773ad7b30ba458bb7dd986b059c67396acf23903979f7b9af123d967R64

Reply:
Yes but is the logic correct?

Reply:
Yes to capture the initial value.

Reply:
So basically if `_last_set_aux_values[i]` is NAN, then it's a major movement? :thinking: